packages/store-sync/src/sqlite/internalTables.ts

```diff
@@ -2,15 +2,18 @@ import { blob, integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
 import { address, asHex, json } from "./columnTypes";
 import { PartialTable } from "./common";
 
-export const chainState = sqliteTable("__chainState", {
+export const CHAIN_STATE_TABLE_NAME = "__chainState";
+export const MUD_STORE_TABLES_NAME = "__mudStoreTables";
```

Comment:
we don't really do uppercase constants like this in MUD

I think we can probably leave the table names where they are (so they're close to their definition) and then do something like

```
export const internalTables = [chainState, mudStoreTables];
export const internalTableNames = internalTables.map(getTableName);
```

Reply:
noted, and updated